codalab/apps/web/views.py

```diff
@@ -1402,14 +1402,14 @@ class SubmissionDelete(LoginRequiredMixin, DeleteView):
     def get_object(self, queryset=None):
         obj = super(SubmissionDelete, self).get_object(queryset)
 
+        is_admin = self.request.user in obj.phase.competition.admins.all()
         self.success_url = reverse("competitions:view", kwargs={"pk": obj.phase.competition.pk})
```

Comment:
Can we move this after the if? So the `is_admin` stuff is grouped, then `self.success_url` is set afterwards?